lib/forwarder.ts

```diff
@@ -13,18 +13,27 @@ import * as crypto from "crypto";
 import { LambdaDestination } from "@aws-cdk/aws-logs-destinations";
 const SUBSCRIPTION_FILTER_PREFIX = "DatadogSubscriptionFilter";
 export function addForwarder(scope: cdk.Construct, lambdaFunctions: lambda.Function[], forwarderARN: string) {
```

Comment:
Nit: I know this was already there, but for abbreviations in camel case like ARN, they should be capitalized like `Arn` instead of `ARN`. Also this is more consistent with other usages of `Arn` in the same file.

Reply:
I actually don't think it matters much which way it is capitalized, but I would at least try to make it consistent within the codebase ideally. Otherwise it could get a bit confusing.